app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
yep. I will change it to .js and give explanation a bit what's coffeescript and why we stick with js.